indico/core/auth.py

```diff
@@ -109,8 +119,14 @@ def handle_auth_error(self, exc, redirect_to_login=False):
         if isinstance(exc, (NoSuchUser, InvalidCredentials)):
             if not getattr(exc, '_indico_no_rate_limit', False):
                 login_rate_limiter.hit()
+                login_rate_limiter_user.hit(exc.identifier)
             logger.warning('Invalid credentials (ip=%s, provider=%s): %s',
                            request.remote_addr, exc.provider.name if exc.provider else None, exc)
+        elif isinstance(exc, InvalidCredentials):
```

Comment:
You can't get into this case because InvalidCredentials is already caught above...